src/index.html

```diff
@@ -1,13 +1,375 @@
+              </div>
+
+              <p class="about__paragraph">
+                A great product is more than what’s in the box. It’s a promise of premium performance, world-class support, and everything you expect from a trusted brand. It’s just one of many reasons why you’ll shop with confidence on Bose.com.
+              </p>
+
+            </div>
+          </div>
+
+        </div>
+      </section>
+
+      <section class="page__section contacts" id="contacts">
+        <div class="container">
+          <h2 class="page__section--title">
+            Contact us
+          </h2>
+
+          <div class="contacts__content">
+            <div class="contacts__content__left">
+              <form
+                action="#"
+                method="GET"
+                class="form"
+                id="form"
+              >
+                <input
+                  class="form-field contacts__input"
+                  type="text"
+                  name="name"
+                  id="name"
+                  placeholder="Name"
+                  required
+                 >
+                <input
+                  type="email"
+                  class="form-field contacts__input"
+                  name="email"
+                  id="email"
+                  placeholder="Email"
+                  required
+                >
+                <textarea
+                  class="form-field form-field--message contacts__input contacts__input-last"
+                  name="message"
+                  placeholder="Message"
+                  id="message"
+                  required
+                ></textarea>
+                <button class="button button--primary">
+                  Send
+                </button>
+              </form>
+            </div>
+
+            <div class="contacts__content__right">
+              <div class="contacts__info">
+                <div class="contacts__title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55"
+                  class="contacts__data"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+              <div class="contacts__info">
+                <div class="contacts__title">
+                  Email
+                </div>
+                <a
+                  href="mailto:hello@miami.com"
+                  class="contacts__data"
+                >
+                hello@miami.com
+                </a>
+              </div>
+              <div class="contacts__info">
+                <div class="contacts__title">
+                  Address
+                </div>
+                <a
+                    href="https://goo.gl/maps/Ds8G6yLW6ohLsjtC9"
+                    class="contacts__data"
+                  >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
```

Comment:
on hover on our address , phone number all elements around it seems like toggle their places please fix it
check this [video](https://www.loom.com/share/295391e58c25483187eb76d071c20e86)

Reply:
actually, I did that because we agreed with Dima to make the hover effect more visible. I removed it now.